File: src/main/resources/static/app/devices/transductors/actuators/entities/ActuatorValues.js
class ActuatorValues {
	
	constructor(date, values) {
		this.date = date;
		this.values = values;
	}
	
	
	getDate() {
		return this.date;
	}
	
	
	getValue(propertyActuated) {
		return this.values[propertyActuated];
	}
	
	
}

File: src/main/java/com/iotassistant/models/devices/ActuatorValues.java
package com.iotassistant.models.devices;

import java.util.Map;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import com.iotassistant.models.devices.transductors.propertyactuated.PropertyActuatedEnum;

@Entity
public class ActuatorValues  extends TransductorValues<PropertyActuatedEnum>{

	@Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int id;
	
	public ActuatorValues() {
		super();
	}



	public ActuatorValues(Map<PropertyActuatedEnum, String> values, String date) {
		super(values, date);
	}
	

}
